types/src/account_config/constants/account_type.rs

```diff
@@ -8,37 +8,37 @@ use move_core_types::{
 };
 use once_cell::sync::Lazy;
 
-static ACCOUNT_TYPE_MODULE_NAME: Lazy<Identifier> =
-    Lazy::new(|| Identifier::new("AccountType").unwrap());
-pub static ACCOUNT_TYPE_MODULE: Lazy<ModuleId> =
-    Lazy::new(|| ModuleId::new(CORE_CODE_ADDRESS, ACCOUNT_TYPE_MODULE_NAME.clone()));
-pub static ACCOUNT_TYPE_STRUCT_NAME: Lazy<Identifier> = Lazy::new(|| Identifier::new("T").unwrap());
```

Comment:
It was using these names to create a VASP account role (empty/unhosted etc.) in the e2e tests.